_layouts/default.html

```diff
@@ -25,11 +25,10 @@ <h1>
 			/>
 		</a>
 		</h1>
-		{% if page.file %}
```

Comment:
We should probably still keep the `if` here for pages that don't have a file.

Reply:
You're right-- as is, this change breaks the home page. :grimacing:

Reply:
Fixed.